src/cbmc/cbmc_parse_options.cpp

```diff
@@ -550,7 +551,10 @@ int cbmc_parse_optionst::doit()
 
   if(cmdline.isset("validate-goto-model"))
   {
-    goto_model.validate(validation_modet::INVARIANT);
+    goto_model.validate(
+      validation_modet::INVARIANT,
+      goto_model_validation_optionst{
+        goto_model_validation_optionst::set_optionst::all_true});
```

Comment:
Given how often that all_true option is used, simply make that the default?

Reply:
Same for INVARIANT?

Reply:
Done. Also for INVARIANT.